bases/ecoindex/cli/app.py

```diff
@@ -37,6 +38,9 @@
 @app.command()
 def analyze(
     url: list[str] = Option(default=None, help="List of urls to analyze"),
+    sitemap: list[str] = Option(
```

Comment:
**❓ question(non-blocking):** Here, do we allow the user to provide multiple sitemap urls ?

Reply:
Hello @vvatelot ! 

I don't know what is your opinion ? 
The Sitemap Spider is supposed to crawl all nested sitemaps in the provided sitemap url. I fear that if we allow severall sitemap urls there will be overlap and it will take too long.

Reply:
I need to modify list[str] to str